### PR TITLE
contact form

### PR DESCRIPTION
**Pull request by solog0039**
In this project
1. GitHub workflow added
2. linters installed
3. contact form completed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -256,9 +256,31 @@ <h3> <img src="image/Rectangle57.png" alt="skill"> Skills </h3>
...
+    <div class="forms">
+      
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+        <form class="upper-form" action="https://formspree.io/f/xoqrrprd" method="post">
+            <input type="text" maxlength="30" required placeholder="Name" class="full-name" name="usrname">
+            <input type="email" required placeholder="Email" class="email" name="usremail">
+            <input type="text" maxlength="500" required placeholder="Write your message here" class="messages"
+                name="usrmessage">
+            <input type="submit" value="Get it touch" class="touch" name="usrsubmit">
+        </form>
```
Comment (koachcode) [inline index.html]: - Please make use of the appropriate font size as it is on the Figma design and center the text so it looks like that of the Figma design. i meant the `<p>` with the text  `I'm always interested in hearing about new projects, so if you'd like to chat please get in touch
- Please the button Get in touch should have the proper padding as it is on the Figma design.
##### Figma Design 
Screenshot from 2022-05-31 21-58-21

##### Your Design 
Screenshot from 2022-05-31 21-58-04

File: index.html [hunk 2]
```diff
@@ -256,9 +256,31 @@ <h3> <img src="image/Rectangle57.png" alt="skill"> Skills </h3>
...
+    </div>
+    <footer class="footer">
+        <hr>
+        <div class="social-links">
+            <img class="github" src="form-image/github.png" alt="github">
+            <img class="linkedin" src="form-image/linkedin.png" alt="linkedIn">
+            <img class="angel" src="form-image/angel.png" alt="angelist">
```
Comment (koachcode) [inline index.html]: - Please use a better icon on the `Linkedin` icon as it looks fade, or you can increase the opacity, if it was initially reduced, so as to look like that of the Figma design.
##### Figma design 
Screenshot from 2022-05-31 22-04-30

##### Your design
Screenshot from 2022-05-31 22-04-13

File: index.html [hunk 3]
```diff
@@ -256,9 +256,31 @@ <h3> <img src="image/Rectangle57.png" alt="skill"> Skills </h3>
...
     </section>
+
+    <div class="forms">
+      
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+        <form class="upper-form" action="https://formspree.io/f/xoqrrprd" method="post">
+            <input type="text" maxlength="30" required placeholder="Name" class="full-name" name="usrname">
+            <input type="email" required placeholder="Email" class="email" name="usremail">
+            <input type="text" maxlength="500" required placeholder="Write your message here" class="messages"
+                name="usrmessage">
```
Comment (koachcode) [inline index.html]: - Please make this a type of `textarea` and not a type of `text`, so as to make your HTML  more semantically correct.

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @solog0039 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

keep-trying-youre-almost-there

 ## Highlights
- Linter all green :+1: 
- Good code structure :100: 
- - Good PR nameing :100: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (solog0039) [reply index.html]: Dear @Ginohmk I can not get LinkedIn icon like in the figma. The icon in the figma is separated in to i and n. so I will keep as it is or please let me know if the is another option

File: index.html (hunk 3 above)
Comment (solog0039) [reply index.html]: Capture

File: index.html (hunk 2 above)
Comment (Kakalanp) [reply index.html]: - If I'm not mistaken, you can select both and [export them](https://help.figma.com/hc/en-us/articles/360040028114-Guide-to-exports-in-Figma) as a png :bulb: 

- Another solution could be to use a service like [icons8](https://icons8.com/) and look for a LinkedIn icon there :smiley:

File: index.html (hunk 1 above)
Comment (Kakalanp) [reply index.html]: You could set your `font-weight` to 34px to fix this issue, that worked on my end :grin:

File: index.html (hunk 3 above)
Comment (Kakalanp) [reply index.html]: I apologize for the mistake, the previous CR meant that you change the HTML tag from `<input>` to [`<textarea>`](https://www.w3schools.com/tags/tag_textarea.asp#:~:text=Definition%20and%20Usage,width%20font%20(usually%20Courier).), as its name states, this is an **area** of **text**, so we should try to use this tag to follow best practices :wink:

**Review by Kakalanp — CHANGES_REQUESTED** [review]
# STATUS: Changes required :exclamation: 

### Hello there @solog0039! hi

### Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. :memo: 

almost there

## Highlights :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:
- Excellent communication with CRs' :monocle_face: 

 ## Required Changes ♻️
***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#Portfolio-mobile-version 
+#Portfolio-mobile-version
```
Comment (danmainah) [inline README.md:1]: - [ ] [OPTIONAL] Kindly check the alignment in your Authors section;
image

**Review by danmainah — APPROVED** [review]
Hi @solog0039 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**